config.go

```diff
@@ -37,6 +38,7 @@ type tgfConfig struct {
 	RecommendedMinimalVersion string
 	RecommendedImage          string
 	Debug                     string
+	FlushCache                string
```

Comment:
Isn't overkill to add this in the config? That means that a user can configure to flush its cache on everyrun through .tgf.config and the parameter store.